ext/date/php_date.c

```diff
@@ -795,8 +795,8 @@ static zend_string *date_format(char *format, size_t format_len, timelib_time *t
 			offset->offset = (t->z);
 			offset->leap_secs = 0;
 			offset->is_dst = 0;
-			offset->abbr = timelib_malloc(9); /* GMT±xxxx\0 */
-			snprintf(offset->abbr, 9, "GMT%c%02d%02d",
+			offset->abbr = timelib_malloc(13); /* GMT±xxxx\0 */
+			snprintf(offset->abbr, 13, "GMT%c%02d%02d",
```

Comment:
Ugh ... can we do something against this?

Reply:
Maybe change the format precision?